### ISSUE TITLE
Show how to integrate backend and front end

### ISSUE THREAD

**Issue by walmon** [hyperledger-labs/convector#15]
Getting Started only shows how to setup a project. There's a reference to a fullstack project, but it is necessary to show the actual details and step by step which are not documented outside the fullstack example.

**Comment by walmon** [comment]
Backend was ready a few weeks ago, working on the frontend.

**Comment by AzzaAN** [comment]
Are you still working on the frontend? Thanks

**Comment by diestrin** [comment]
@AzzaAN correct, we're preparing a release with really good news for front-end devs

**Comment by koptan** [comment]
@diestrin  any update on front-end release ?

**Comment by giacom0c** [comment]
@walmon @diestrin Anything about the front-end?
